src/MonoTorrent.BEncoding/MonoTorrent.BEncoding.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Condition="$(TargetFramework) == 'netstandard2.0'" Include="System.Memory" Version="4.5.4" />
+    <PackageReference Condition="$(TargetFramework) == 'netstandard2.0'" Include="System.Memory" Version="4.5.0" />
```

Comment:
4.5.0 *or newer* is fine for MonoTorrent when targeting .NET 4.7.2, 4.8.0 or 4.8.1.